src/lib/merkle_ledger/syncable_intf.ml

```diff
@@ -23,6 +23,8 @@ module type S = sig
 
   val set_all_accounts_rooted_at_exn : t -> addr -> account list -> unit
 
+  (* Get all of the accounts that is in a subtree of the underlying merkle tree where it's root is `address`. 
```

Comment:
"its" :-)

Make a docstring with `(**`

Reply:
Please fix this before merging.